### PR TITLE
Model publishing checklist JS binding after core

### PR DESCRIPTION
**Pull request by davisshaver**
Modeled after https://github.com/WordPress/WordPress/blob/master/wp-admin/js/inline-edit-post.js#L50-L54

See https://github.com/fusioneng/publishing-checklist/issues/4#issuecomment-115261120

### CODE REVIEW

File: assets/js/src/publishing-checklist.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 (function($){
 
-	$('.publishing-checklist-show-list').on('click', function() {
+	$('#the-list').on('click', '.publishing-checklist-show-list', function(){
```
Comment (danielbachhuber) [inline assets/js/src/publishing-checklist.js]: This will break on Edit Posts, so we'll need to accommodate both contexts

**Comment by davisshaver** [comment]
@danielbachhuber review/merge? Handles both Manage and Edit with hideChecklist and showChecklist functions.

File: assets/js/src/publishing-checklist.js [hunk 2]
```diff
@@ -1,15 +1,23 @@
 (function($){
 
-	$('.publishing-checklist-show-list').on('click', function() {
+	$('#the-list').on('click', '.publishing-checklist-show-list', showChecklist );
+
+	$('#the-list').on('click', '.publishing-checklist-hide-list', hideChecklist );
+
+	$('.publishing-checklist').on('click', '.publishing-checklist-show-list', showChecklist );
```
Comment (danielbachhuber) [inline assets/js/src/publishing-checklist.js]: Can we scope this binding to the Publish submit box? Otherwise, it will get bound twice on Manage Posts

**Comment by danielbachhuber** [comment]
:+1:

**Pull request merged**